src/%IPM/Main.cls

```diff
@@ -1002,9 +1037,9 @@ ClassMethod Init(ByRef pCommandInfo) [ Internal ]
 		Set tOldSourceControlClass = ""
 	}
 	
-	If (tOldSourceControlClass '= "") && (tOldSourceControlClass '= tCompositeSCClass) && (tOldSourceControlClass '= tPackageMgrSCClass) {
-		Write !,"Enabling package manager extension... "
-		$$$ThrowOnError(##class(%IPM.ExtensionBase.Composite).SetExtensionClasses(tOldSourceControlClass,$ListBuild(tPackageMgrSCClass)))
+	If (tOldSourceControlClass '= "") && (tOldSourceControlClass '= tCompositeSCClass) && (tOldSourceControlClass '= tServerMenuExtClass) {
```

Comment:
Should we be tracking `tOldServerMenuExtClass` as well? Here you are checking its equality against `tOldSourceControlClass` which doesn't make much sense.